core_backend/app/contents/models.py

```diff
@@ -14,10 +13,10 @@
 from sqlalchemy.ext.asyncio import AsyncSession
 from sqlalchemy.orm import Mapped, mapped_column
 
-from ..config import LITELLM_API_KEY, LITELLM_ENDPOINT, LITELLM_MODEL_EMBEDDING
-from ..contents.config import PGVECTOR_VECTOR_SIZE
 from ..models import Base, JSONDict
 from ..schemas import FeedbackSentiment
+from ..utils import embedding
```

Comment:
Since `embedding` is used by both `content` and `urgency_rules`, I factored it out.